src/tutorials/pupdate.pug

```diff
@@ -0,0 +1,139 @@
+extends ./tutorial_layout.pug
+
+block title
+    | Interactive Tutorial | Beam Developers
+
+block tutorialTitle
+    h1 Learn how to send progress updates.
+
+block tutorialContent
+    h2#intro Introduction
+    p.
+        When using interactive, you can send global "Progress Updates" that will
+        trigger events for every user that currently has the interactive app
+        open(are viewing the page.) By doing this you can trigger things such
+        as cooldowns that work for everyone at the same time, or you can
+        show what tactile is being pressed or the persentage of users
+        that are pushing it. You could also show what the general direction
+        that a joystick would be aiming.
+
+    h2#controlEditor Types of Progress Updates.
+    p.
+        There are various types of progress updates, or rather progress updates
+        for different things. Such as Joystick updates of tactile updates.
+        You can also do screen updates if you are using screen controls.
+        Below will be a list of progress update tags, for more info go to the
+        #[a(href='/reference/interactive/index.html') reference guide].
+
+    table.table
```

Comment:
Its been on my list for quite awhile. I'll find some time at some point, hopefully today.